enterprise/internal/gitserver/integration_tests/commits_test.go

```diff
@@ -67,7 +67,7 @@ func TestGetCommits(t *testing.T) {
 			nil,
 		}
 
-		commits, err := gitserver.NewTestClient(http.DefaultClient, inttests.GitserverAddresses).GetCommits(ctx, getTestSubRepoPermsChecker("file1", "file3"), repoCommits, true)
+		commits, err := gitserver.NewTestClient(http.DefaultClient, inttests.GitserverAddresses, nil).GetCommits(ctx, getTestSubRepoPermsChecker("file1", "file3"), repoCommits, true)
```

Comment:
In all of these callsites, I believe it's better to pass around the default gRPC getter func rather than just a nil. 

```suggestion
		commits, err := gitserver.NewTestClient(http.DefaultClient, inttests.GitserverAddresses, defaultGRPCSource).GetCommits(ctx, getTestSubRepoPermsChecker("file1", "file3"), repoCommits, true)
```